### PR TITLE
Add redirects for the docs paths and fix the redirects for the root paths

### PR DESCRIPTION
**Pull request by hatched**

### CODE REVIEW

File: webapp/blueprint.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import os
+import urllib.parse
```
Comment (mitechie) [inline webapp/blueprint.py]: make life easy? 

    from urllib.parse import urljoin

File: webapp/blueprint.py [hunk 2]
```diff
@@ -31,6 +32,11 @@ def root():
         return flask.redirect(JAAS_URL)
 
 
+@gui.route("/docs/<path:path>")
+def docs(path=""):
+    return flask.redirect(urllib.parse.urljoin("https://docs.jujucharms.com", path))
```
Comment (mitechie) [inline webapp/blueprint.py]: will this hit any other redirects that help with the version to non-versioned urls as well?

File: webapp/blueprint.py [hunk 3]
```diff
@@ -48,7 +54,10 @@ def root():
 @gui.route("/store")
 @gui.route("/support")
 def jaas():
-    return flask.redirect(JAAS_URL)
+    if flask.request.path == "/home":
+        return flask.redirect(JAAS_URL)
+    else:
+        return flask.redirect(urllib.parse.urljoin(JAAS_URL, flask.request.path))
```
Comment (mitechie) [inline webapp/blueprint.py]: are there any tests in the codebase around the urls that we care about to where they end up going?

Comment (hatched) [reply webapp/blueprint.py]: There aren't, we should add them though - I'll create a follow-up issue for this

File: webapp/blueprint.py (hunk 1 above)
Comment (hatched) [reply webapp/blueprint.py]: done thanks

**Pull request merged**